users/src/main/java/io/crate/http/netty/HttpAuthUpstreamHandler.java

```diff
@@ -98,6 +97,9 @@ private void handleHttpChunk(ChannelHandlerContext ctx, MessageEvent e) {
 
     @VisibleForTesting
     static void sendUnauthorized(Channel channel, @Nullable String body) {
+        if (LOGGER.isWarnEnabled() && body != null) {
```

Comment:
body is `null` if the handler receives a HttpChunk but is not authorized. IMO it is ok to not send an additional message in the HTTP response body